.ci/build.bat

```diff
@@ -0,0 +1,12 @@
+echo "Executing gradle"
+
+call gradlew shadowJar -PVERSION=%APPVEYOR_REPO_TAG_NAME%
```

Comment:
Do we want to explicitly run the `check` task as well here as is done in build.sh?